aksel.nav.no/website/pages/templates/soknad-introside-aap/soknad-introside-aap.tsx

```diff
@@ -0,0 +1,175 @@
+                </List.Item>
+              </List>
+            </Box>
+            <Box>
+              <Accordion>
+                <Accordion.Item>
+                  <Accordion.Header>
+                    Informasjon vi henter om deg
+                  </Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det så informasjon om hvor vi vil hente
+                      opplysninger om søkeren og hva slags opplysninger vi
+                      henter.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>
+                    Hvordan vi behandler personopplysninger
+                  </Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det stå informasjon om hvordan vi behandler
+                      personopplysningene til søkeren.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>Automatis behandling</Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det stå informasjon om hva automatisk behandling
+                      er for noe og hva det betyr for søkeren. Samt informasjon
+                      om søkers rettigheter ved automatisk avslag.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>Vi lagrer svar underveis</Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det stå informasjon om hvordan denne søknaden
+                      mellomlagrer informasjonen til søkeren og hvor lenge
+                      informasjonen lagres. Både automatisk lagring og samtykke
+                      til lagring med lagre-knapp.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+              </Accordion>
+            </Box>
+            <Box paddingBlock="0 12">
+              <BodyLong>
+                Det er viktig at du gir oss riktige opplysninger slik at vi kan
+                behandle saken din.{" "}
+                <Link href="#gi-riktige-opplysninger">
+                  Les mer om viktigheten av å gi riktige opplysninger.
+                </Link>
+              </BodyLong>
+              <Box paddingBlock="4 8">
+                <Checkbox>
+                  Jeg vil svare så godt jeg kan på spørsmålene i søknaden.
+                </Checkbox>
+              </Box>
+              <Button
+                variant="primary"
+                icon={<ArrowRightIcon />}
```

Comment:
```suggestion
                icon={<ArrowRightIcon aria-hidden />}
```
`aria-hidden` burde være på alle ikonene i button (virker ikke som voiceover leser opp ikonet nå, men kan hende det er forskjellig i andre skjermlesere)